test/tchannel.js

```diff
@@ -18,6 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+var net = require('net');
```

Comment:
nope, will clean that out too.

On Thu, Aug 20, 2015 at 2:12 PM, Jeff Wolski <notifications@github.com>
wrote:

> In test/tchannel.js
> <https://github.com/uber/tchannel/pull/1174#discussion_r37521859>:
>
> > @@ -18,6 +18,8 @@
> >  // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
> >  // THE SOFTWARE.
> >
> > +var net = require('net');
>
> is this necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uber/tchannel/pull/1174/files#r37521859>.
>